2020-02-Spring/0_arrays_matrix/maximum-subarray/maxSubArray.java

```diff
@@ -0,0 +1,22 @@
+class Solution {
```

Comment:
Could use some tests here... if you have time.

Also, if I were to clone this and compile the class as-is, I wouldn't be able to due to the discrepancy between the file name and the class name. (Code that doesn't work out of the box is annoying and not the best look.)